### ISSUE TITLE
Suggestion: Folder support

### ISSUE THREAD

**Issue by DPS2004** [jtothebell/fake-08#125]
Folders would make organization of games easier, such as putting all games in a series together, or all files of a multicart.

**Comment by jtothebell** [comment]
Yeah, this would definitely be helpful for large cart libraries and multicart games. The app should already have the ability to load a cart from an arbitrary location if it is passed as an argument, but obviously the default interface doesn't show those directories. So that will have to be updated, and I think there would have to be some other work to support loading multicart carts from a directory.
